16/umbraco-cms/customizing/foundation/fetching-data/custom-generated-client.md

```diff
@@ -0,0 +1,99 @@
+---
+description: Learn how to create a custom generated client with TypeScript types for your OpenAPI specification.
+---
+
+# Custom Generated Client
+
+Umbraco uses [@hey-api/openapi-ts](https://heyapi.dev/openapi-ts/get-started) to generate its HTTP client for the OpenAPI specification of the Management API. It is available through the `@umbraco-cms/backoffice/http-client` package.
+
+{% content-ref url="http-client.md" %}
+[http-client.md](http-client.md)
```

Comment:
```suggestion
[Umbraco HTTP Client](http-client.md)
```

Friendlier link label?

Reply:
It's a content-ref that will pull in a card view of the article, so don't think this matters, but thanks for noticing :)